.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,24 +11,21 @@
 
 ## Type of Changes
 <!-- What type of changes does your code introduce? -->
-<!-- Put an `[x]` in any of the boxes that apply: -->
+<!-- (Remove inapplicable items) -->
 
-- [ ] Bug fix _(non-breaking change which fixes an issue)_
-- [ ] New feature _(non-breaking change which adds functionality)_
-- [ ] Breaking change _(fix or feature that changes existing functionality)_
+- Bug fix _(non-breaking change which fixes an issue)_
+- New feature _(non-breaking change which adds functionality)_
+- Breaking change _(fix or feature that changes existing functionality)_
 
 ## Checklist
-<!-- Verify the following points and put an `[x]` in the boxes that apply: -->
+<!-- Verify the following points before submitting the pull request. -->
 
-- [ ] Builds & tests completed successfully (`./gradlew test integrationTest`).
-- [ ] My code follows the code style of this project.
+- I have signed-off my commits per http://www.dita-ot.org/DCO.
+- Builds & tests completed successfully (`./gradlew test integrationTest`).
+- My code follows the code style of this project.
     -  <https://github.com/dita-ot/dita-ot/wiki/Java-Coding-Conventions>
     -  <https://github.com/dita-ot/dita-ot/wiki/XSLT-Coding-Conventions>
-- [ ] I have updated the unit tests to reflect the changes in my code.
-- [ ] My change requires a change to the documentation. @dita-ot/docs
-- [ ] I will update the documentation accordingly or create a new [docs][] issue.
-
-<!-- At mention person or team responsible for reviewing proposed changes. -->
+- I have updated the unit tests to reflect the changes in my code.
+- I will update the documentation accordingly or create a new [docs][] issue.
```

Comment:
I'm not sure about this. It's important, but this is the person making a promise of something they will do in the future, instead of saying they've already checked everything. I don't have a suggestion how to improve it, but somehow this jumps out.